scripts/setup.sh

```diff
@@ -43,6 +43,11 @@ function maybeInstallDependencies {
       shift
       ;;
 
+      --suite=*)
```

Comment:
I understand that block 26-50 is designed to parse only --skip-install and --run-minified-tests.

But what I don't get is the externally/internally thing -- sometimes I run run_e2e_tests.sh with the arguments --sharding=true --sharding-instances=2, for example. If it fails for --suite, shouldn't it fail for those as well?